src/main/java/net/discordjug/javabot/util/WebhookUtil.java

```diff
@@ -97,11 +97,21 @@ public static CompletableFuture<ReadonlyMessage> mirrorMessageToWebhook(@NotNull
 				.submit()
 				.exceptionally(e -> null)//if the member cannot be found, use no member information
 				.thenCompose(member -> 
-					mirrorMessageToWebhook(webhook, originalMessage, newMessageContent, threadId, components, embeds, member));
+					mirrorMessageToWebhook(webhook, originalMessage, newMessageContent, threadId, components, null, embeds, member));
```

Comment:
Please don't unnecessary duplicate code. You can just replace the whole method body here with a call to the other overload similar to the following:
```java
public static CompletableFuture<ReadonlyMessage> mirrorMessageToWebhook(@NotNull Webhook webhook, @NotNull Message originalMessage, String newMessageContent, long threadId, @Nullable List<LayoutComponent> components, @Nullable List<MessageEmbed> embeds) {
	mirrorMessageToWebhook(webhook, originalMessage, newMessageContent, threadId, components, null, embeds);
```
or similar